src/backend/TrafficCourts/Workflow.Service/Consumers/DisputeUpdateRequestAcceptedConsumer.cs

```diff
@@ -108,7 +112,22 @@ public async Task Consume(ConsumeContext<DisputeUpdateRequestAccepted> context)
                         }
                         break;
                     case DisputeUpdateRequestUpdateType.DISPUTANT_DOCUMENT:
-                        // TODO: update document metadata set StaffReviewStatus to Accepted
+                        // update all documents in this patch to ACCEPTED
+                        if (patch.UploadedDocuments is not null) {
+                            foreach (UploadDocumentRequest uploadDocumentRequest in patch.UploadedDocuments) {
+                                Guid? documentId = uploadDocumentRequest.DocumentId;
+                                if (documentId is not null) {
+                                    // extract current document properties
+                                    Coms.Client.File file = await _documentService.GetFileAsync(documentId.Value, context.CancellationToken);

```

Comment:
I am going to double check this. I dont think we should need to load the entire file just to add a tag. To save meta data,  you have to read and write the entire file.

Reply:
Yeah, that's a good call. I just pushed an update where I removed this file retrieval and instead just used a new `documentService.SetTagsAsync()` to update a specific tag without the need to first fetch the file.